src/core/sync/condition.d

```diff
@@ -74,7 +74,7 @@ class Condition
      * Throws:
      *  SyncError on error.
      */
-    this( Mutex m ) nothrow @safe
+    this( Mutex m ) nothrow @trusted
```

Comment:
The whole function shouldn't be `@trusted`. Instead, the part that needs to escape `@safety` should use `@trusted` delegates.